### PR TITLE
Resolve #419 by adding emoticon notes in CiviMail and Token docs abou…

### LINKED ISSUE #419: Tokens and click-through
There should be a notice/warning in the token section that using tokens in links for CiviMail will bypass click-through tracking.  I got burned by this in a serious way, and this isn't mentioned in the token section nor in the click-through CiviMail section.  Maybe the notice/warning should be in both sections.

### PR DESCRIPTION
**Pull request by seamuslee001**
…t url tracking implications of personalised links

### CODE REVIEW

File: docs/common-workflows/tokens-and-mail-merge.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ information.
 !!! caution
     For security purposes, checksum links only last for seven days, starting from the time the email is sent. To change this duration, go to **Administer > System Settings > Misc (Undelete, PDFs, Limits, Logging, Captcha, etc.)**.
 
+!!! note
+    Checksum urls will not be tracked within CiviMail url tracking as they are personalised to the recipient.
```
Comment (homotechsual) [inline docs/common-workflows/tokens-and-mail-merge.md]:
```suggestion
    Checksum URLs will not be tracked within CiviMail URL tracking as they are personalised to the recipient.
```

File: docs/email/mass-mailings-using-civimail.md [hunk 2]
```diff
@@ -259,6 +259,11 @@ two options described next.
         with long links containing the name of your site and a long code 
         looking like this http://example.com/sites/all/modules/civicrm/extern/url.php?u=529&qid=29011.
 
+    !!! note "Personalised Tokens"
+        If you include a checksum or any kind of personalised token in your url
```
Comment (homotechsual) [inline docs/email/mass-mailings-using-civimail.md]:
```suggestion
        If you include a checksum or any kind of personalised token in your URL 
```

File: docs/email/mass-mailings-using-civimail.md [hunk 3]
```diff
@@ -259,6 +259,11 @@ two options described next.
         with long links containing the name of your site and a long code 
         looking like this http://example.com/sites/all/modules/civicrm/extern/url.php?u=529&qid=29011.
 
+    !!! note "Personalised Tokens"
+        If you include a checksum or any kind of personalised token in your url 
+        e.g. "http://example.org/civicrm/mailing/view?reset=1&id={mailing.key}&cid={contact.contact_id}&{contact.checksum}"
```
Comment (homotechsual) [inline docs/email/mass-mailings-using-civimail.md]:
```suggestion
        e.g. `http://example.org/civicrm/mailing/view?reset=1&id={mailing.key}&cid={contact.contact_id}&{contact.checksum}`
```

File: docs/email/mass-mailings-using-civimail.md [hunk 4]
```diff
@@ -259,6 +259,11 @@ two options described next.
...
+        Such urls will not be redirected so as able to be tracked / counted.
```
Comment (homotechsual) [inline docs/email/mass-mailings-using-civimail.md]:
```suggestion
        Such URLs will not be redirected so are not able to be tracked / counted.
```

**Review by homotechsual — CHANGES_REQUESTED** [review]

**Comment by seamuslee001** [comment]
@MikeyMJCO have finally got around to making the changes now for you

**Pull request merged**